code/game/objects/items/grenades/clusterbuster.dm

```diff
@@ -1,191 +1,191 @@
-////////////////////
```

Comment:
The line endings got fucked up on this file, which editor are you using?

Reply:
> 
> 
> The line endings got fucked up on this file, which editor are you using?

I been using visual studio code, I don't know why my line endings are fucked. I never even touched them, despite that. I was able to run the game fine without any issues.

Reply:
either your editor or your git client is forcing line endings to change.

Reply:
I guess so, I mean either way it doesn't screw over the code and I am able to run the game fine.